configurator/src/main.rs

```diff
@@ -350,9 +408,89 @@ fn main() -> Result<(), anyhow::Error> {
                         masked: false,
                     },
                 },
+                rpc: match config.rpc.enabled {
+                    true => Some(RpcProperties {
+                        quick_connect_url: Property::String {
+                            value: format!(
+                                "clightning-rpc://{}:{}@{}:{}",
+                                config.rpc.user, config.rpc.password, peer_tor_address, 8080
+                            ),
+                            description: Some(
+                                "A convenient way to connect a wallet to a remote node".to_owned(),
+                            ),
+                            copyable: true,
+                            qr: true,
+                            masked: true,
+                        },
+                        rpc_username: Property::String {
+                            value: format!("{}", config.rpc.user,),
+                            description: Some("Username for RPC connections".to_owned()),
+                            copyable: true,
+                            qr: false,
+                            masked: true,
+                        },
+                        rpc_password: Property::String {
+                            value: format!("{}", config.rpc.password,),
+                            description: Some("Password for RPC connections".to_owned()),
```

Comment:
Don't use to_owned here: parameterize the structure over a lifetime and use an &'static ref to the strings.